test/simulator/main/org/apache/cassandra/simulator/paxos/PaxosSimulation.java

```diff
@@ -72,7 +72,7 @@ public abstract class PaxosSimulation implements Simulation, ClusterActionListen
 
     private static String createDescription(int[] primaryKeys, int id, String idString)
     {
-        return primaryKeys.length == 1 ? Integer.toString(primaryKeys[0]) : Arrays.toString(primaryKeys) + "/" + id + ": " + idString;
+        return (primaryKeys.length == 1 ? Integer.toString(primaryKeys[0]) : Arrays.toString(primaryKeys)) + "/" + id + ": " + idString;
```

Comment:
not sure if it was intentional or not, but looking at the logs and seeing `-4` as the only description was confusing... the previous logic would only add `id` and `idString` if-and-only-if `partitionKeys.length != 1`